templates/solutions/telco/5g-core.html

```diff
@@ -0,0 +1,595 @@
+              <p class="u-text--muted">
+                As a proud member of the Cloud Native Computing Foundation (CNCF), Canonical uses Kubernetes to manage Containerised Network Functions (CNFs) in telecom networks. Kubernetes is the leading container orchestration platform and automates deployment, scaling, and operations.
+              </p>
+            </div>
+          </div>
+          <hr class="p-rule--muted" />
+          <div class="p-section--shallow">
+            <div class="row">
+              <div class="col-2">
+                <h4 class="p-text--small-caps">What's included</h4>
+              </div>
+              <div class="col-4">
+                <ul class="p-list--divided">
+                  <li class="p-list__item is-ticked">Cloud native</li>
+                  <li class="p-list__item is-ticked">Automated scaling</li>
+                  <li class="p-list__item is-ticked">Self-healing</li>
+                  <li class="p-list__item is-ticked">Rolling updates</li>
+                </ul>
+              </div>
+            </div>
+          </div>
+          <hr class="p-rule--muted" />
+          <div class="row">
+            <p>
+              <a href="/engage/hpe-telco-validated-design-with-canonical-kubernetes">Discover how HPE and Canonical's Kubernetes Telco validated design can future-proof your network&nbsp;&rsaquo;</a>
+            </p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="p-section">
+    <div class="row--50-50">
+      <hr />
+      <div class="col">
+        <h2>Canonical's 5G Core architecture</h2>
+      </div>
+      <div class="col">
+        <div class="p-image-container">
+          {{ image(url="https://assets.ubuntu.com/v1/9d96ea3a-5gcore.png",
+                    alt="5G Core architecture",
+                    width="1200",
+                    height="676",
+                    hi_def=True,
+                    loading="auto",
+                    attrs={"class": "p-image-container__image"}) | safe
+          }}
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="p-section">
+    <div class="row">
+      <div class="p-section--shallow">
+        <hr />
+        <h2 class="p-text--small-caps">Leading in open source and cloud-native transformation</h2>
+      </div>
+    </div>
+    <div class="u-fixed-width">
+      <div class="p-logo-section">
+        <div class="p-logo-section__items">
+          <div class="p-logo-section__item">
+            {{ image(url="https://assets.ubuntu.com/v1/0dfd3923-onf-logo.png",
+                        alt="ONF",
+                        width="157",
+                        height="256",
+                        hi_def=True,
+                        loading="lazy",
+                        attrs={"class": "p-logo-section__logo"}) | safe
+            }}
+            <hr class="p-rule--muted" />
+            <p>
+              <a href="https://opennetworking.org/">Learn more&nbsp;&rsaquo;</a>
```

Comment:
Descriptive CTAs are more accessible, wee should apply similar spans for each
```suggestion
              <a href="https://opennetworking.org/">Learn more<span class=u-off-screen> about ONF</span>&nbsp;&rsaquo;</a>
```